src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,36 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [];
+
+  constructor(name) {
+    this.health = 100;
+    this.name = name;
+    this.hidden = false;
+    Animal.alive.push(this);
+  }
 }
 
 class Herbivore extends Animal {
```

Comment:
You should use also use a constructor method to inherit values, e.g:

```
constructor(name, health) {
    super(name, health);

    this.hidden = false;
  }
```

Reply:
Thanks for comments. Please check code if is ok. Thanks.